avocado/core/requirements/resolver.py

```diff
@@ -0,0 +1,30 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: Red Hat Inc. 2021
+# Authors: Willian Rampazzo <willianr@redhat.com>
+
+from ..nrunner import Runnable
```

Comment:
There is no strong argument, just following what we already have under `core`, but I think we can use absolute imports here.

Reply:
okay, here is the "strong" argument:

$ selftests/modules-boundaries.sh
* Checking for avocado.core imports from examples/tests: 0
* Checking for non-relative avocado imports from avocado/core: 4
avocado/core/requirements/resolver.py:from avocado.core.nrunner import Runnable
avocado/core/runners/requirement_package.py:from avocado.utils.software_manager.main import MESSAGES
avocado/core/runners/requirement_package.py:from avocado.utils.software_manager.manager import SoftwareManager
avocado/core/runners/requirement_package.py:from avocado.core import nrunner
* Checking for avocado imports from avocado/utils: 0
ERROR: 4 module boundary infringements found

Reply:
I think that those checks are important, but we have to fix them. IIUC we have `check for non-relative avocado imports from avocado/core` because inside core we don't want to have imports out of core and that is valid reason. But this check doesn't do that, because it is possible to create relative import to a different package. So we have to change it but not remove it.